app/views/content/welcome/my-journey-into-teaching/_introduction.html.erb

```diff
@@ -1,36 +1,176 @@
+      because it has one of the best educational systems in the world.
+    </p>
 
-  <h2>I’ve worked out how to get every student interested in maths</h2>
+    <h2>I’ve worked out how to get every student interested in maths</h2>
 
-  <p>
-    Some kids will say &quot;I can’t do algebra&quot; or &quot;I don’t like numbers.&quot;
-    And to be honest, I know how that feels. I&apos;m definitely more a fan of Pure
-    maths than other topics. I try to find out what they’re interested in – whether
-    it’s card games, football or whatever – and use this to make maths relevant.
-  </p>
+    <p>
+      Some kids will say &quot;I can’t do algebra&quot; or &quot;I don’t like numbers&quot;.
+      And to be honest, I know how that feels. I&apos;m definitely more a fan of pure
+      maths than other topics. I try to find out what they’re interested in – whether
+      it’s card games, football or whatever – and use this to make maths relevant.
+    </p>
 
-  <div class="featured">
-    <h2>This year I'm looking forward to&hellip;</h2>
+    <h2>What may surprise you?</h2>
+
+    <p>
+      I didn’t always love maths – it wasn’t until later in secondary school
+      that I discovered my potential.
+    </p>
+
+    <h2>It’s not always easy</h2>
+
+    <p>
+      The relationship that you have with the students is so special, and I did
+      struggle with remote learning during lockdown as it was harder to connect
+      with them. I missed them making jokes!
+    </p>
+
+    <div class="featured">
+      <h2>This year I'm looking forward to&hellip;</h2>
+
+      <figure>
+        <blockquote>Witnessing the progress of my students. I have such a great relationship with them and can't wait to spend more time in the classroom again.</blockquote>
+      </figure>
+    </div>
+  <% elsif subject_category == "science" %>
+    <h2>It all started with my own experiences</h2>
+
+    <p>
+      I really enjoyed the coaching aspect of the degree and getting to see the
+      kids improve. After uni, I did a few odd jobs from bathroom selling to
+      recruitment. But nothing had that ‘feel good factor.’ But then I took a
+      cover job in a local school and I fell in love with teaching.
+    </p>
+
+    <h2>The move from basketball to biology was easier than I thought</h2>
+
+    <p>
+      When the assistant head suggested teaching science, I wasn’t sure I was cut
+      out for it, I’d never been that academic myself. It turned out my degree
+      covered a lot of the science curriculum, so I decided to give it a go. I’ve
+      never looked back.
+    </p>
+
+    <h2>I discovered that science is all about making connections</h2>
+
+    <p>
+      I’m always showing my students how science relates to everything they’re
+      learning. From enzymes in cooking to the surprising history of a lot of
+      medical discoveries. I love it when they ask a question that I’ve never
+      even thought of.
+    </p>
```

Comment:
Additional text to add in for Holly:

What may surprise you?
I didn’t always want to teach. When I first graduated, I was determined not to go back into education – I didn’t think I would enjoy the setting.

It’s not always easy
I think I’m firm but fair. You have to be to establish a strong relationship with the class where they respect you.

This year it’s all about developing new skills
I’m doing training to make sure I can support the kids both practically and emotionally this year. I want to build their confidence.